specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -2266,6 +2474,123 @@
       ],
       "description": "The List hybrid machine operation response."
     },
+    "LicenseProfile": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/LicenseProfileProperties",
+          "description": "Hybrid Compute License Profile properties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Describes a license profile in a hybrid machine."
+    },
+    "LicenseProfileUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/LicenseProfileUpdateProperties",
+          "description": "Describes License Profile Update Properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceUpdate"
+        }
+      ],
+      "description": "Describes a Machine Extension Update."
+    },
+    "EsuKey": {
+      "type": "object",
+      "properties": {
+        "sku": {
+          "type": "string",
+          "description": "SKU number."
+        },
+        "licenseStatus": {
+          "type": "string",
+          "enum":[
+            "1",
+            "0"
+          ],
+          "description": "The current status of the license profile key."
+        }
+      }
+    },
+    "EsuStatus": {
+      "type": "string",
+      "enum":[
+        "enrolled",
+        "unenrolled",
+        "ineligible"
+      ],
+      "description": "The ESU enrollment state."
+    },
+    "EsuStatusUpdateRecord": {
+      "type": "object",
+      "properties": {
+        "oldState": {
+          "$ref": "#/definitions/EsuStatus",
+          "description": "The old ESU enrollment state."
+        },
+        "newState": {
+          "$ref": "#/definitions/EsuStatus",
+          "description": "The new ESU enrollment state."
+        },
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time that the state has been changed."
+        }
+      }
+    },
+    "LicenseProfileProperties": {
+      "type": "object",
+      "properties": {
+        "arcEsuEnrolled": {
+          "type": "string",
+          "enum": [
+            "true",
+            "false"
+          ],
+          "description": "Specifies wether the machine has enrolled in ESU or not."
+        },
+        "esuKeys": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EsuKey"
+          },
+          "description": "The list of ESU keys."
+        },
+        "esuHistory": {
```

Comment:
lets call this esuEnrollmentHistory so its clearer what its for. don't want customers to think it's supposed to be the history of the ESUs that have been made available/applied